sysid-application/src/main/native/cpp/analysis/JSONConverter.cpp

```diff
@@ -117,18 +119,17 @@ std::string sysid::ToCSV(std::string_view path, wpi::Logger& logger) {
     throw std::runtime_error("Unable to write to: " + loc);
   }
 
-  outputFile << "Timestamp (s),Test,";
+  fmt::print(outputFile, "Timestamp (s),Test,");
   if (type == analysis::kDrivetrain || type == analysis::kDrivetrainAngular) {
-    outputFile << "Left Volts (V),Right Volts (V),"
-               << "Left Position (" << abbreviation << "),"
-               << "Right Position(" << abbreviation << "),"
-               << "Left Velocity (" << unit << "/s),"
-               << "Right Velocity (" << unit << "/s),"
-               << "Gyro Position (deg),Gyro Rate (deg/s)";
-
+    fmt::print(
+        outputFile,
+        "Left Volts (V),Right Volts (V),Left Position ({0}),Right "
+        "Position ({0}),Left Velocity ({0}/s),Right Velocity ({0}/s),Gyro "
+        "Position (deg),Gyro Rate (deg/s)\n",
+        abbreviation);
```

Comment:
Did you mean to use unit rather than abbreviation for some of these?

Reply:
No, they should all be abbreviation; not sure why it was unit before. Abbreviation is used for velocity in the second case as well.